packages/devtools/lib/src/ui/primer.dart

```diff
@@ -119,7 +119,10 @@ class PTabNav extends CoreElement {
     }
 
     for (PTabNavTab tab in tabs) {
-      tab.click(() => selectTab(tab));
+      tab.click(() {
+        selectTab(tab);
+        tab.performOnClick();
```

Comment:
why is
performOnClick() needed in addition to selectTab(tab)?

Reply:
selectTab just highlights the selected tab in the tab nav. I basically need on tab selected functionality. I can subclass PTabNav and override selectTab instead.